### PR TITLE
NIAD-379: Acceptance type 2 transaction - first

### PR DESCRIPTION
**Pull request by grzegorz-robak**

### CODE REVIEW

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory-with-nhs.edifact.ignore.dat [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+S01+1'
+RFF+TN:1'
+NAD+GP+2750922,295:900'
+HEA+ACD+A:ZZZ'
+HEA+ATP+2:ZZZ'
+DTM+956:19920114:102'
+S02+2'
+PNA+PAT+N/10/10:OPI+++SU:KENNEDY'
+PDI+2'
+NAD+PAT++??:26 FARMSIDE CLOSE:ST PAULS CRAY:ORPINGTON:KENT'
```
Comment (bartek-sarul) [inline src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory-with-nhs.edifact.ignore.dat]: ??

Comment (bartek-sarul) [reply src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory-with-nhs.edifact.ignore.dat]: ok, I see this in json

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory-with-nhs.fhir.ignore.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          {
+            "identifier": {
+              "system": "https://fhir.hl7.org.uk/Id/gmc-number",
+              "value": "2750922,295"
+            }
+          }
+        ],
+        "identifier": [
+          {
+            "system": "https://fhir.nhs.uk/Id/nhs-number",
```
Comment (bartek-sarul) [inline src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory-with-nhs.fhir.ignore.json:40]: not mandatory

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory-with-nhs.fhir.ignore.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (bartek-sarul) [inline src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory-with-nhs.fhir.ignore.json]: just remove the file
we will create it when doing optionals

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory.fhir.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+              "ORPINGTON",
+              "KENT"
+            ],
+            "use": "home"
+          }
+        ],
+        "extension": [
+          {
+            "url": "http://hl7.org/fhir/StructureDefinition/patient-birthPlace",
+            "valueString": "GLASGOW"
```
Comment (bartek-sarul) [inline src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory.fhir.json]: not matndatory atm

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory-with-nhs.fhir.ignore.json (hunk 3 above)
Comment (grzegorz-robak) [reply src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory-with-nhs.fhir.ignore.json]: ok

File: src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory.fhir.json (hunk 4 above)
Comment (grzegorz-robak) [reply src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory.fhir.json]: ok

**Review by bartek-sarul — DISMISSED** [review]

**Review by mattd-kainos — APPROVED** [review]

**Review by jaroslaw-wojciechowski — APPROVED** [review]

**Pull request merged**